release_notes.md

```diff
@@ -1 +1 @@
-
+- add pprof support [#50](https://github.com/xmidt-org/themis/pull/50)
```

Comment:
We can delete this file as we're no longer using it. We should move this entry to changelog.md

Reply:
Is there anywhere this process is documented?

Reply:
I don't think we have it documented yet (I might be wrong, @kristinaspring, thoughts?). We've just been enforcing it in PRs.
But anyways, if it's not there, should it live in the "Pull Requests" section of the Contributing.md file?

Reply:
Yeah.  We might want to file an issue in the appropriate repo, if the docs don't exist yet.